packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -367,18 +374,7 @@ export class SharedTree
 	protected override async loadCore(services: IChannelStorageService): Promise<void> {
 		await super.loadCore(services);
 		this.checkout.setTipRevisionForLoadedData(this.trunkHeadRevision);
-		this._events.emit("afterBatch");
-	}
-
-	protected override submitCommit(
```

Comment:
We no longer need to worry about filtering our calls to `submitCommit` because the event that triggers submission is registered on the main branch, but _not_ any transaction branches that fork off of the main branch.